### PR TITLE
Add pull request instructions

### PR DESCRIPTION
**Pull request by sarahseewhy**
### What

Adds instructions for creating pull requests:

* Link to the GDS Way guide to PRs
* Add guidance for PR title length
* Add PR description example format

Also adds a sign-post to the "how to propose new tasks" workflow diagram for new joiners.

### Why

Better instructions will help us avoid unclear PRs which cause us to lose context on (important) past decisions. Moreover, we code in the open and should follow best practices for creating open source code.

### CODE REVIEW

File: source/applications/developing.html.md.erb [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
 ---
 
 # Get started with the applications
 
-GovWifi applications are written in [Ruby](https://www.ruby-lang.org/en/) using either [Rails](https://rubyonrails.org/) and [Sinatra](http://sinatrarb.com/) web frameworks.
+GovWifi applications are written in [Ruby](https://www.ruby-lang.org/en/) using either [Rails](https://rubyonrails.org/) and [Sinatra](http://sinatrarb.com/) web frameworks. Our [infrastructure](https://github.com/alphagov/govwifi-terraform) is written in Terraform.
 
 To get started you'll need Ruby installed on your system. We recommend using a Ruby version manager like [rbenv](https://github.com/rbenv/rbenv) which will allow you to install the correct version of Ruby for GovWifi projects.
 
+We also recommend using the Terraform version manager  [tfenv](https://github.com/tfutils/tfenv) to ensure we work with the same Terraform version.
```
Comment (bethwalton) [inline source/applications/developing.html.md.erb:14]: Rogue double space?

File: source/ways-of-working/index.html.md.erb [hunk 2]
```diff
@@ -28,7 +28,7 @@ We work in quarters and fortnightly sprints.
 
 We have a [product roadmap](https://docs.google.com/spreadsheets/d/1sHVJgnOoUe-vk01h6Uzthj1669_vJfZiSCdIJgUCyNU/edit?ts=5f7c59ce#gid=1927198713) which sets out what we’re planning to work on.
 
-We use Trello to track our progress. We have separate boards for [planning](https://trello.com/b/upKC2Xrg/govwifi-planning-ideas) and for our [sprints](https://trello.com/b/9jIWLDIa/govwifi-stand-up-q3-2020).
+We use Trello to track our progress. We have separate boards for [planning](https://trello.com/b/upKC2Xrg/govwifi-planning-ideas) and for our [sprints](https://trello.com/b/9jIWLDIa/govwifi-stand-up-q3-2020). There's [a helpful workflow diagram for how to propose new work](https://docs.google.com/drawings/d/1giP8P3g_XC7L4RJdo4NctPNmqwPF9pRq7Kmk-wSiRKY/edit).
```
Comment (bethwalton) [inline source/ways-of-working/index.html.md.erb:31]: Can we change to 'new technical work'?

File: source/applications/developing.html.md.erb (hunk 1 above)
Comment (sarahseewhy) [reply source/applications/developing.html.md.erb:14]: Good spot! I'll fix that.

File: source/ways-of-working/index.html.md.erb (hunk 2 above)
Comment (sarahseewhy) [reply source/ways-of-working/index.html.md.erb:31]: Sure!

**Review by bethwalton — APPROVED** [review]
Looks good from a content perspective :)

**Pull request merged**